.DEREK.yml

```diff
@@ -1,13 +0,0 @@
-curators:
-  - pchico83
-  - rberrelleza
-  - rlamana
-  - ifbyol
-  - jLopezbarb
-
-features:
-  - dco_check
```

Comment:
Maybe we should keep this for some time instead of forcing the protection rule. Before enabling the protection rule I would like to check that automatic bots that sends PRs to the repo use verified commits. For example, would the backport flow keep working once we merge and configure the protection rule? Is that bot creating a signed commit?